src/elements/content-sidebar/activity-feed/comment/RepliesToggle.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import PlainButton from '../../../../components/plain-button';
+
+import './RepliesToggle.scss';
+
+type Props = {
+    isAlwaysExpanded?: boolean,
+    onHideReplies: (index: number) => void,
+    onShowReplies: () => void,
+    repliesShownCount: number,
+    repliesTotalCount: number,
+};
+
+const RepliesToggle = ({
+    isAlwaysExpanded = false,
+    onShowReplies,
+    onHideReplies,
+    repliesShownCount,
+    repliesTotalCount,
+}: Props) => {
+    if (isAlwaysExpanded || repliesTotalCount <= 1) {
+        return null;
+    }
+
+    const hasMoreRepliesToShow = repliesTotalCount > repliesShownCount;
+    const toggleMessage = hasMoreRepliesToShow ? messages.showReplies : messages.hideReplies;
+    const repliesToLoadCount = Math.max(repliesTotalCount - repliesShownCount, 0);
```

Comment:
Is there a case where repliesTotalCount can actually be less than the repliesShownCount? Also if that value is 0 should we even be showing the toggle message?

Reply:
There shouldn't be a case where total count is less than shown count. And if there is, the `repliesTotalCount <= 1` conditional will take care of it. We have a test to capture that case as well, and the toggle is not shown

Reply:
We had some questions about how we want to deal with this in our BDD session but we can just leave it as be for now.